### DISCUSSION TITLE
Which Minecraft?

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/teals-minecraft#23] [General]
Is this going to be lessons in Minecraft Education?

Can Microsoft give us free licenses?

Will this be a curriculum we can use in our schools?

**Comment by hollasch** [comment]
At this point, I don't know about the status of TEALS and any commitment to the Minecraft project. You should ask your regional manager directly about status and plans for current support.

With regard to licenses, no license is needed for the Minecraft project, as servers are open source.
